spec/factories/vacancies.rb

```diff
@@ -200,9 +204,5 @@
       external_reference { "J3D1" }
       external_advert_url { "https://example.com/jobs/123" }
     end
-
-    trait :non_sequential_job_title do
-      job_title { factory_sample(job_titles) }
-    end
```

Comment:
This was my previous attempt not to call `sequence` in production. I thought it worked, but the error came back today.